merk/src/proofs/query/mod.rs

```diff
@@ -1,10 +1,6 @@
 mod map;
 
-use std::{
-    cmp::{max, min, Ordering},
-    collections::BTreeSet,
-    ops::{Range, RangeFrom, RangeFull, RangeInclusive, RangeTo, RangeToInclusive},
-};
+use std::{cmp::{max, min, Ordering}, cmp, collections::BTreeSet, ops::{Range, RangeFrom, RangeFull, RangeInclusive, RangeTo, RangeToInclusive}};
```

Comment:
nit: imo this is worse 🤷

Reply:
hmm a single `cargo fmt` will bring it back to normal